go.mod

```diff
@@ -31,7 +27,7 @@ require (
 	github.com/gorilla/mux v1.7.1 // indirect
 	github.com/gorilla/websocket v1.4.0 // indirect
 	github.com/gravitational/configure v0.0.0-20180808141939-c3428bd84c23
-	github.com/gravitational/coordinate v0.0.0-20200227044100-12af3c0f9593
+	github.com/gravitational/coordinate v0.0.0-20200506112554-9ae5851333fd
```

Comment:
With the new release, can we drop the pseudoversion and used the tagged version?